/opt/rocm-6.2.1/libexec/rocprofiler/tblextr.py: hcc ops data not found: '6341841428852472:6341841428899832 4:0 CopyHostToDevice:5:2103669', 5, 2103669
/opt/rocm-6.2.1/libexec/rocprofiler/tblextr.py: hcc ops data not found: '6341841428525791:6341841428647552 3:0 CopyHostToDevice:5:2103667', 5, 2103667
Traceback (most recent call last):
  File "/opt/rocm-6.2.1/libexec/rocprofiler/tblextr.py", line 1130, in <module>
    ops_filtr = fill_ops_db("OPS", "COPY", db, indir)
  File "/opt/rocm-6.2.1/libexec/rocprofiler/tblextr.py", line 929, in fill_ops_db
    kernel_table_handle = db.add_table(kernel_table_name, ops_table_descr)
  File "/opt/rocm-6.2.1/libexec/rocprofiler/sqlitedb.py", line 52, in add_table
    cursor.execute(stm)
sqlite3.OperationalError: table OPS already exists
Traceback (most recent call last):
  File "/opt/rocm-6.2.1/libexec/rocprofiler/tblextr.py", line 1130, in <module>
Traceback (most recent call last):
  File "/opt/rocm-6.2.1/libexec/rocprofiler/tblextr.py", line 1130, in <module>
Traceback (most recent call last):
  File "/opt/rocm-6.2.1/libexec/rocprofiler/tblextr.py", line 1130, in <module>
    ops_filtr = fill_ops_db("OPS", "COPY", db, indir)
  File "/opt/rocm-6.2.1/libexec/rocprofiler/tblextr.py", line 929, in fill_ops_db
Traceback (most recent call last):
  File "/opt/rocm-6.2.1/libexec/rocprofiler/tblextr.py", line 1130, in <module>
    ops_filtr = fill_ops_db("OPS", "COPY", db, indir)
  File "/opt/rocm-6.2.1/libexec/rocprofiler/tblextr.py", line 929, in fill_ops_db
    ops_filtr = fill_ops_db("OPS", "COPY", db, indir)
  File "/opt/rocm-6.2.1/libexec/rocprofiler/tblextr.py", line 929, in fill_ops_db
    kernel_table_handle = db.add_table(kernel_table_name, ops_table_descr)
  File "/opt/rocm-6.2.1/libexec/rocprofiler/sqlitedb.py", line 52, in add_table
    kernel_table_handle = db.add_table(kernel_table_name, ops_table_descr)
  File "/opt/rocm-6.2.1/libexec/rocprofiler/sqlitedb.py", line 52, in add_table
    ops_filtr = fill_ops_db("OPS", "COPY", db, indir)
  File "/opt/rocm-6.2.1/libexec/rocprofiler/tblextr.py", line 929, in fill_ops_db
    cursor.execute(stm)
sqlite3.OperationalError: table OPS already exists
    cursor.execute(stm)
sqlite3.OperationalError: table OPS already exists
    kernel_table_handle = db.add_table(kernel_table_name, ops_table_descr)
  File "/opt/rocm-6.2.1/libexec/rocprofiler/sqlitedb.py", line 52, in add_table
    cursor.execute(stm)
sqlite3.OperationalError: table OPS already exists
    kernel_table_handle = db.add_table(kernel_table_name, ops_table_descr)
  File "/opt/rocm-6.2.1/libexec/rocprofiler/sqlitedb.py", line 52, in add_table
    cursor.execute(stm)
sqlite3.OperationalError: table OPS already exists
/opt/rocm-6.2.1/libexec/rocprofiler/tblextr.py: hcc ops data not found: '6341841430201314:6341841430248834 2:0 CopyHostToDevice:5:2103678', 5, 2103678
srun: error: t006-004: tasks 0-7: Exited with exit code 1
